engine/dlib/src/dlib/buffer.cpp

```diff
@@ -616,4 +621,70 @@ namespace dmBuffer
         buffer->m_ContentVersion++;
         return RESULT_OK;
     }
+
+
+    Result UpdateBounds(HBuffer hbuffer, dmhash_t stream_name) {
```

Comment:
We said we should calculate and set this from the mesh component.

Reply:
yup we did :-). 

I tried that first but there were to many dependencies on private parts of the code in buffer.cpp and moved this to buffer.cpp to make it work.

Let me try this again and discuss not that things get clearer.

Reply:
Ok, here is the obstacle i decided to move this (UpdateBounds()) to buffer.cpp instead of comp_mesh.cpp for:

`GetBuffer()` requires a `g_BufferContext` to be able to get the internal buffer from an HBuffer. This is private stuff only available in buffer.cpp.

Thoughts ?

Reply:
nevermind. I moved UpdateBounds() to buffer.cpp. Used GetStrean() without problems.